### DISCUSSION TITLE
how to set icon name for window?

### DISCUSSION THREAD

**Discussion by ITwrx** [can-lehmann/owlkettle#68] [Q&A] [answered]
My app doesn't show an icon in the xfce workspace switcher's pager (panel's applicationmenu plugin?). I added a 48x48 to the /usr/share/icons/hicolor/48x48/apps dir but i still get the default window icon. How to define my apps window icon?

https://docs.gtk.org/gtk4/method.Window.set_icon_name.html

**Comment by can-lehmann** [comment, marked as answer]
I added support for `Window.iconName` here: https://github.com/can-lehmann/owlkettle/commit/072e78e022c39cec9f5163ecaf40208784a1e129

**Reply by ITwrx** [reply]
sweet! thanks!
